app/views/devise/registrations/edit.html.erb

```diff
@@ -1,8 +1,13 @@
 <h2>Edit <%= resource_name.to_s.humanize %></h2>
-
+<%= current_user.admin? %>
```

Comment:
use could edit own information, so not sure why this condition is needed

Reply:
removed junk check